test/index.test.js

```diff
@@ -327,12 +327,12 @@ describe('JWT token decoding', function () {
   it('should complain if the HTTP Authorization header is in the wrong format', async function () {
     const response = await server.inject({ method: 'GET', url: '/decode', headers: { Authorization: 'FOO' } })
 
-    expect(response.statusCode).toEqual(400)
+    expect(response.statusCode).toEqual(401)
     expect(response.json()).toEqual({
-      code: 'FST_JWT_BAD_REQUEST',
-      statusCode: 400,
-      error: 'Bad Request',
-      message: 'Format is Authorization: Bearer [token]'
+      code: 'FST_JWT_NO_AUTHORIZATION_IN_HEADER',
```

Comment:
This feels like oauth2-mock-server matured its error messages but I need to dig into to make sure.  Hence the draft.